README.md

```diff
@@ -5,13 +5,6 @@
 
 A wonderfully simple way to load your code.
 
-Tired of futzing around with `require` statements everywhere, littering your code
```

Comment:
I actually liked this straight and direct introductory paragraph. Maybe not move it all, since it is still easier to load files with require_all? No?

Reply:
Maybe jus remove the part of "regardless of the dependency structure" and leave other part as it was?